Add Beneficiaries Component

CajetanRodrigues opened the pull request:
Fix #19 

### Beneficiaries Component

**Balsamiq Design**

[https://openmf.mybalsamiq.com/projects/onlinebankingapp30/BeneficiaryComponent](url)

![beneficiary_balsa](https://user-images.githubusercontent.com/37682760/62455751-b159b580-b794-11e9-9814-fc577d1581a5.PNG)

### **UI**
Beneficiaries Component
 
Search Facility
----------------------------------
 
User can Search through all the records
 
Sorting Facility
-----------------------------------
 
User can sort the records w.r.t to each field
 
Pagination
------------------------------------
 
User can choose how many records are to be displayed at a time.


![beneficiary](https://user-images.githubusercontent.com/37682760/62455836-d8b08280-b794-11e9-868f-2d61d47c9ee4.PNG)

ankit01ojha commented:
@CajetanRodrigues squash these commits into one.

ankit01ojha reviewed (commented):
@CajetanRodrigues , could you just squash these commits into one.

CajetanRodrigues commented:
> @CajetanRodrigues , could you just squash these commits into one.

Done

CajetanRodrigues committed:
Add beneficiary table with sorting & sticky column feature


add pagination to table


add css styling to table


add media queries for responsiveness


update beneficiary service


update beneficiary service


add pagination,sorting,search facility


add beneficiary component
